doc/api/packages.md

```diff
@@ -10,87 +10,247 @@ A folder containing a `package.json` file, and all subfolders below that folder
 until the next folder containing another `package.json` file, are a _package
 scope_.
 
-## Determining module system
+## `package.json` supported fields
```

Comment:
This should maybe go at the bottom? It’s more like an index; all of these fields should have been covered already by their respective sections. And the content for each field should just be a stub, maybe describing the field in a sentence but then linking to the relevant section.